tests/jwt_disable_flag_tests.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-undef */
+import { expect } from 'chai';
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import 'mocha';
+import request from 'supertest';
+import jwtProxy, { JwtProxyOptions } from '../src/index';
+import { genericHandlers } from './handlers2';
+import { after } from 'mocha';
+
+
+describe.skip('Disable flag tests.', () => {
```

Comment:
I had to disable these tests as process.env.VAR is not working withing the scope of each test, the describe blocks, etc. when I was setting it the actual middleware code showed the process.env.JWTP_DISABLE as "undefined" although i was setting it in the test code.  This could just be an issue that we cannot test without mocking process.env.

Reply:
But with jwt_verify_env_tests.ts did not run into this issue?

Reply:
Not sure if this is something you'd want to try, but I enjoy using as much "dotenv" as possible, including in mocha tests... https://github.com/motdotla/dotenv/issues/127

It might help to get `process.env` set-up for your tests.